data/card_catalogue/social.json

```diff
@@ -70,10 +70,10 @@
     "tier": "blue",
     "details": {
       "rules": {
-        "active": "Speak with any dead NPC, then draw another blue.",
+        "active": "Speak with any dead NPC. Their voice can be heard by all nearby NPCs.",
         "passive": null
       },
-      "flavour": "How'd you get this number? I didn't even own a phone when I was alive!",
```

Comment:
Not a strong opinion, but I'm thinking we should stray away from flavour texts that reference other flavour text. If you have both cards it can be funny, but if you just get one then its a disappointment.